src/main/webapp/app/iris/settings/iris-settings-update/iris-common-sub-settings-update/iris-common-sub-settings-update.component.ts

```diff
@@ -52,11 +54,18 @@ export class IrisCommonSubSettingsUpdateComponent implements OnInit, OnChanges {
     // Button types
     WARNING = ButtonType.WARNING;
     // Icons
-    faTrash = faTrash;
+    readonly faTrash = faTrash;
+    readonly faQuestionCircle = faQuestionCircle;
+    readonly faCircleExclamation = faCircleExclamation;
 
     protected readonly IrisSubSettings = IrisSubSettings;
     protected readonly IrisSubSettingsType = IrisSubSettingsType;
 
+    protected readonly eventTranslationKeys = {
+        [IrisEventType.BUILD_FAILED]: 'artemisApp.iris.settings.subSettings.proactivityBuildFailedEventEnabled.label',
+        [IrisEventType.PROGRESS_STALLED]: 'artemisApp.iris.settings.subSettings.proactivityProgressStalledEventEnabled.label',
+    };
+
     constructor(
         accountService: AccountService,
         private irisSettingsService: IrisSettingsService,
```

Comment:
Could you adapt this file to the new client guidelines?

Reply:
There is currently a separate PR for migrating Iris components according to latest coding guidelines for angular.

Please see #9615